v0.3/tutorials/quickstart counter.eve

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Quickstart
+---
+
+{% raw %}
+
+# Quickstart + Counter
+
+This guide expands on the quickstart by adding a simple incrementing counter to the `Counter` page.
+
+## Quickstart code
```

Comment:
I would also link out rather than re-embed. The examples should be able to stand on their own or be embedded into the quickstart.